clang/test/CodeGen/aarch64-elf-pauthabi.c

```diff
@@ -0,0 +1,59 @@
+// RUN: %clang_cc1 -triple aarch64-linux -S -emit-llvm -o - \
```

Comment:
`-S` should be removed. I've cleaned up the whole clang/test suite.

After https://github.com/llvm/llvm-project/pull/91140 this will lead to an error.

Reply:
Fixed, thanks, see 37a4d9d08fefd8c42c21b92ad2e3c1fa07fac126